### PR TITLE
BHBC-1654: Public page treatments

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview

## Links to jira tickets

https://quartech.atlassian.net/browse/BHBC-1654

### CODE REVIEW

File: app/src/pages/public/PublicProjectPage.tsx [hunk 1]
```diff
@@ -35,32 +85,88 @@ const PublicProjectPage = () => {
   const restorationTrackerApi = useRestorationTrackerApi();
   const classes = useStyles();
 
-  const codes = useCodes();
```
Comment (NickPhura) [inline app/src/pages/public/PublicProjectPage.tsx:38]: We can replace the useEffect and useStates for fetching codes with this call to `useCodes`.

(Just for this file, I have a separate PR open to move the other files over: https://github.com/bcgov/restoration-tracker/pull/49).

File: api/src/paths/public/project/{projectId}/treatments/year/list.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { HTTP400 } from '../../../../../../errors/custom-error';
+
+import { getLogger } from '../../../../../../utils/logger';
+import { getAPIUserDBConnection } from '../../../../../../database/db';
+import { TreatmentService } from '../../../../../../services/treatment-service';
+
+const defaultLog = getLogger('/api/project/{projectId}/treatments/year/list');
```
Comment (KjartanE) [inline api/src/paths/public/project/{projectId}/treatments/year/list.ts:9]: should have the "/public/" in the logger path

File: api/src/paths/public/project/{projectId}/treatments/year/list.ts [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+      const treatmentService = new TreatmentService(connection);
+
+      const data = await treatmentService.getProjectTreatmentsYears(projectId);
+
+      await connection.commit();
+
+      return res.status(200).json(data);
+    } catch (error) {
+      defaultLog.error({ label: 'getProjectTreatments', message: 'error', error });
```
Comment (KjartanE) [inline api/src/paths/public/project/{projectId}/treatments/year/list.ts:77]: label is out dated, "getPublicProjectTreatmentYears"

**Review by KjartanE — APPROVED** [review]
I think it looks good!

**Pull request merged**